.travis.yml

```diff
@@ -1,29 +1,39 @@
+dist: xenial
+sudo: required
+
 language: php
 
+cache:
+    directories:
+        - $HOME/.composer/cache/files
+
 matrix:
+    fast_finish: true
     include:
         - php: 5.6
         - php: 7.0
         - php: 7.1
           env: CHECK_CS=true
+        - php: 7.2
+        - php: 7.3
 
 # test only master (+ Pull requests)
```

Comment:
Not "only master" anymore given change below, right?

Reply:
indeed, fixed in 
a948ba1